### PR TITLE
Fix contrib webserver so that it compiles.

### PR DESCRIPTION
**Pull request by brendandburns**
Also, add a test for external connectivity.

@dchen1107 this should have caught your recent issue.

### CODE REVIEW

File: test/e2e/network.go [hunk 1]
```diff
@@ -34,6 +35,17 @@ func TestNetwork(c *client.Client) bool {
...
 	}
 
+	// Test basic external connectivity.
+	resp, err := http.Get("http://google.com/")
+	if err != nil {
+		glog.Errorf("unable to talk to the external internet: %v", err)
+		return false
+	}
+	if resp.StatusCode != http.StatusOK {
+		glog.Errorf("unexpected error code: %v (%v)", resp.StatusCode, resp)
```
Comment (a-robinson) [inline test/e2e/network.go]: may want to clarify - unexpected error code talking to the external internet

**Comment by a-robinson** [comment]
LGTM, just one nit.

**Comment by brendandburns** [comment]
done.

**Comment by satnam6502** [comment]
If we push a new image, can it be given a unique tag (:1.0, :1.1 etc.) and then can the uses of this image be updated to use the tagged version of the image?

**Comment by brendandburns** [comment]
wasn't planning on pushing a new image, just noticed that it wasn't compiling, so I fixed it...

**Comment by a-robinson** [comment]
Shippable seems to be stuck, so I'm just going ahead and merging this.

**Pull request merged**